translations/messages.en.xlf

```diff
@@ -2169,5 +2169,17 @@
         <target>Unlocked</target>
       </segment>
     </unit>
+    <unit id="jz2qkbb" name="general.phrase.no-content-found">
+      <segment>
+        <source>general.phrase.no-content-found</source>
+        <target>No content found</target>
+      </segment>
+    </unit>
+    <unit id="FM8B.gO" name="general.phrase.empty-database">
```

Comment:
Is this used anywhere?

Reply:
Yup, that's here:
https://github.com/bolt/themes/blob/71b5850e3e2fae0214b1ecd6fa152a9b0043f81e/base-2018/partials/_no_content.twig